src/screens/epoch/utils.ts

```diff
@@ -0,0 +1,91 @@
+import numeral from 'numeral';
+import {
+  InflationType,
+  InflationGovernorType,
+  EpochScheduleType,
+} from './types';
+
+export const inflationData = (data: InflationType, t: any) => {
+  return ([
+    {
+      name: t('epoch'),
+      value: data ? data.epoch : '',
+      detail: t('epochDetail'),
+    },
+    {
+      name: t('totalInflation'),
+      value: data ? numeral(data.total).format('0.0[000]') : '',
+      detail: t('totalInflationDetail'),
+    },
+    {
+      name: t('validatorInflation'),
+      value: data ? numeral(data.validator).format('0.0[000]') : '',
+      detail: t('validatorInflationDetail'),
+    },
+    {
+      name: t('foundationInflation'),
+      value: data ? numeral(data.foundation).format('0.0[000]') : '',
+      detail: t('foundationInflationDetail'),
+    },
+  ]);
+};
```

Comment:
3. I updated to '-' because in my perspective, 0 is confusing while the data coming back might be 0.

Reply:
What if the data really is 0

Reply:
If the data is really 0 then it will show 0. Like this `value: data ? data.initial : '-',`

Reply:
1. Updated.

Reply:
2. The reason for '0.0[000]' was because I thought it look better than '0.0[00]'. But now I updated it to the same format style with title bar inflation rate. Less messy following same style.